pages/AdListPage/[id]/index.js

```diff
@@ -40,24 +40,33 @@ export default function AdDetailsPage() {
           alt={`Examplephoto of ${ad.adTitle}`}
         />
       </ImageContainer>
-      <h3>{ad.adTitle}</h3>
+      <AdTitle>{ad.adTitle}</AdTitle>
       <Description>
         <p>Task description:</p>
         {ad.adDescription}
       </Description>
       <Attributes>
-        <p>category: {ad.category}</p>
+        <p>category:</p>
+        <CategoryItem> {ad.category}</CategoryItem>
         <p>Tags: </p>
         <TagsList>
           {ad.tags.map((tag, index) => {
-            return <li key={index}> {tag} </li>;
+            return <TagItem key={index}> {tag} </TagItem>;
```

Comment:
If the tags are unique, you could also use the `tag` as a `key` instead of the index.